package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@zoralabs/nft-components",
-  "version": "0.1.12",
+  "version": "0.1.13-pre1",
```

Comment:
This looks good – although can this line be removed?

Reply:
should be up to date now @iainnash